content/core-features/authentication.textile

```diff
@@ -211,6 +211,53 @@ ably.auth.requestToken({ clientId: 'client@example.com' }, function(err, tokenDe
 
 *Note:* The authenticity of the JWT *will not* be checked, due to Ably not having access to your SECRET key.
 
+h4(#jwt-claims). Using JWT for authenticated user claims
+
+It is possible to use JWTs to authenticate claims for users in your channels, allowing or disallowing certain interactions.
+
+Messages can include trusted fields in the `extras` object that contain information copied from their authentication token, making it unique to each user. These fields are deemed trusted as their content is added is supplied directly from Ably and not the client. The trusted field is a channel-specific piece of information and needs to be set for other channels if similar permissions are desired.
+
+*Note:* This does not apply to presence or metadata messages.
+
+An example use case is when using "Message Interactions":/realtime/messages#message-interactions you might want to create a moderator in a chat channel that has the ability to delete sent messages. When the moderator sends an interaction request to delete a message Ably checks that the user's claim for a value moderator for that channel before actioning their request.
```

Comment:
An example use case is when using "Message Interactions":/realtime/messages#message-interactions. You might want to use trusted claims to define 'moderator' users in a chat channel, who have the ability to delete any sent messages. When the moderator sends an interaction to mark messages in a channel as deleted, your application should check that the user's claims to verify they are a moderator for that channel before actioning their request.